<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MTHIRUNA-LAP" name="com.opc.test.SSO_LoginTest" tests="1" failures="0" timestamp="8 Feb 2013 10:25:09 GMT" time="27.730" errors="1">
  <testcase name="test" time="27.730" classname="com.opc.test.SSO_LoginTest">
    <error type="java.lang.AssertionError" message="Unable to find element with xpath == //input[@id=&#039;sso_username&#039;] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 20.16 seconds
Build info: version: &#039;2.25.0&#039;, revision: &#039;17482&#039;, time: &#039;2012-07-18 21:08:56&#039;
System info: os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.6.0_38&#039;
Driver info: driver.version: RemoteWebDriver
Session ID: a7012e78-7e72-4e50-87fc-23bb8388247e">
      <![CDATA[java.lang.AssertionError: Unable to find element with xpath == //input[@id='sso_username'] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 20.16 seconds
Build info: version: '2.25.0', revision: '17482', time: '2012-07-18 21:08:56'
System info: os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.6.0_38'
Driver info: driver.version: RemoteWebDriver
Session ID: a7012e78-7e72-4e50-87fc-23bb8388247e
	at org.testng.Assert.fail(Assert.java:94)
	at com.opc.test.SSO_LoginTest.test(SSO_LoginTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- test -->
</testsuite> <!-- com.opc.test.SSO_LoginTest -->
